src/style.css

```diff
@@ -1,3 +1,73 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.header {
+  margin: 0;
+  display: flex;
+  justify-content: space-between;
+  /* background-color: rgb(118, 118, 118); */
+  align-items: center;
+}
+
+.nav {
+  display: inline-block;
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  list-style: none;
+
+}
+
+.nav__link {
+  position: relative;
+  color: #000;
```

Comment:
It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)

Reply:
what do  mean

Reply:
Move all repeatable colors to the CSS variables

Reply:
please do the same with this color as you did with `--for-links`